kernel/enter.c

```diff
@@ -648,9 +652,7 @@ int myst_enter_kernel(myst_kernel_args_t* args)
     ECHECK(_create_main_thread(args->event, args->cwd, &thread));
     __myst_main_thread = thread;
 
-    thread->main.umask = MYST_DEFAULT_UMASK;
```

Comment:
Is the deletion of umask intentional?

Reply:
This seems to be a duplicate of [L409](https://github.com/deislabs/mystikos/blob/main/kernel/enter.c#L409).